LICENSE

```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
+
+Copyright (c) 2022, Max Planck Gesellschaft
```

Comment:
I think since the package was created last year already, the year here should better be 2021.

Reply:
year is updated